### ISSUE TITLE
Créer un script HALO/HAHO

### ISSUE THREAD

**Issue by yanek** [11rdp/modpack#16]
- Possibilité de marcher dans la soute de l'avion (C130, lequel ?)
- L'avion doit être en mouvement
- ? 

Quelqu'un aurait une vidéo d'un saut HALO ? Que je vois à quoi ça ressemble dans le vrai mon réel de la réalité véritable.

**Comment by Sociohunter** [comment]
http://le.cos.free.fr/chuteurs.htm
Here you go. La vidéo montre bien à quoi ça ressemble, et les explications plutôt bien faites pour les techniques de saut (de l'armée française).
Edit : (les belges ont les mêmes techniques de saut que nous, puisqu'on les entraîne je crois, d'où la vidéo montrant des FS belges)

**Comment by 11RDP-LoupVert** [comment]
C'est destiné au SP et réseau local en l'état mais il y a une version serveur dédié *pourrait* rentrer dans les plans de l'auteur :
https://forums.bistudio.com/forums/topic/191056-leap-halo-parajump-operations/

----

Je pense qu'on utilisait le script de Poker (http://forums.bistudio.com/showthread.php?157793-ATM-Airdrop-A3-Beta) depuis un C-130 -statique- ([http://11rdp.fr/forum/index.php?topic=241.msg1595]) duquel on s'élencait pour les SOTGH et les dérives sous voile.

A propos de la possibilité de marcher dans un véhicule en mouvement, ce serait impossible à cause d'une limitation intrinsèque du moteur de jeu et çà ne semble pas avoir changé : https://github.com/acemod/ACE3/issues/3209

**Comment by yanek** [comment]
Effectivement, il était possible de marcher dans la soute du Ebu's C130 (en mouvement, je ne suis pas convaincu, mais en vol et statique, oui), à condition de boucher les trous dans les textures avec d'autres objets comme des caisses de munitions cachées dans la carlingue. Mais ça impliquerait de briser la règle du _"aucune dépendance en dehors de ACE et CBA"_.

Et en même temps, je vois pas trop quoi utiliser d'autre qu'un C130. J'ai vraiment du mal à imaginer comment intégrer une solution complète pour le saut HALO dans le pack. Je peux créer des outils, ou des effets pour le rendre plus immersif, mais je pense que pour les interactions véhicules, et le saut en lui-même, il faudra s'en remettre aux scripts mission.

**Comment by Nikkaaa** [comment]
J'vais pas avoir énormément de temps ces prochains jours, mais j'pensais à un simple module de vol d'un cargo avec une éjection et une ouverture de voile automatique ou non (configurable). 

La seule question qui demeure sans réponses de mon côté c'est la gestion du décollage, si quelqu'un a des idées ?

**Comment by 11RDP-LoupVert** [comment]
Peut-être adapter un mod de voile auto ? https://forums.bistudio.com/forums/topic/181542-static-line-parachute-mod/
Pour le décollage, je suppose qu'il s'agit de se faire emmener par l'IA à la hauteur voulue à une coordonnée  déterminée (par un clic sur la carte?)?

**Comment by Nikkaaa** [comment]
Je pensais plutôt à un module d'entrée configurable (type d'avions, hauteur, type de saut etc..) et un module de sortie qui sera l'emplacement du drop.

Le seul truc qui sort complétement de mon domaine de compétence c'est comment faire décoller l'avion ? Est-ce que sur une map custom cela va pas poser problème ? Comment définir à quel moment l'avion doit décoller sans passer par un action menu ou aucune autre interaction IG.

**Comment by 11RDP-LoupVert** [comment]
![](http://i.imgur.com/iesHEgH.jpg)

L'addon LOW and  EXTREME  ALTITUDE  PARAJUMPS de Von Quest* semble fonctionner en multi : https://www.youtube.com/watch?v=kqThJoi3Zak



* https://forums.bistudio.com/forums/topic/191056-leap-halo-parajump-operations/

https://www.youtube.com/watch?v=KdqVedVJcck
https://www.youtube.com/watch?v=EB4aUr2f5_0
